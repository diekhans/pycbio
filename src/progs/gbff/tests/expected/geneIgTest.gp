IGLC1	NT_011520	+	2628125	2628443	2628125	2628443	1	2628125	2628443
IGLC2	NT_011520	+	2633726	2634079	2633726	2634044	1	2633726	2634079
IGLC3	NT_011520	+	2639082	2639400	2639082	2639400	1	2639082	2639400
IGLC7	NT_011520	+	2655336	2655654	2655336	2655654	1	2655336	2655654
IGLJ1	NT_011520	+	2626529	2626567	2626529	2626567	1	2626529	2626567
IGLJ2	NT_011520	+	2632366	2632404	2632366	2632404	1	2632366	2632404
IGLJ3	NT_011520	+	2637736	2637774	2637736	2637774	1	2637736	2637774
IGLJ7	NT_011520	+	2654138	2654176	2654138	2654176	1	2654138	2654176
IGLV1-36	NT_011520	+	2177076	2177372	2177076	2177372	1	2177076	2177372
IGLV1-40	NT_011520	+	2154884	2155183	2154884	2155183	1	2154884	2155183
IGLV1-44	NT_011520	+	2125989	2126285	2125989	2126285	1	2125989	2126285
IGLV1-47	NT_011520	+	2102883	2103179	2102883	2103179	1	2102883	2103179
IGLV1-50	NT_011520	+	2072444	2072743	2072444	2072743	1	2072444	2072743
IGLV1-51	NT_011520	+	2067598	2067894	2067598	2067894	1	2067598	2067894
IGLV10-54	NT_011520	+	1959934	1960230	1959934	1960230	1	1959934	1960230
IGLV11-55	NT_011520	+	1946812	1947124	1946812	1947124	1	1946812	1947124
IGLV2-11	NT_011520	+	2525764	2526061	2525764	2526061	1	2525764	2526061
IGLV2-14	NT_011520	+	2491972	2492269	2491972	2492269	1	2491972	2492269
IGLV2-18	NT_011520	+	2467849	2468146	2467849	2468146	1	2467849	2468146
IGLV2-23	NT_011520	+	2431178	2431475	2431178	2431475	1	2431178	2431475
IGLV2-33	NT_011520	+	2321431	2321728	2321431	2321728	1	2321431	2321728
IGLV2-8	NT_011520	+	2556055	2556352	2556055	2556352	1	2556055	2556352
IGLV3-1	NT_011520	+	2613856	2614141	2613856	2614141	1	2613856	2614141
IGLV3-10	NT_011520	+	2545057	2545347	2545057	2545347	1	2545057	2545347
IGLV3-12	NT_011520	+	2505354	2505644	2505354	2505644	1	2505354	2505644
IGLV3-16	NT_011520	+	2480687	2480977	2480687	2480977	1	2480687	2480977
IGLV3-19	NT_011520	+	2453919	2454209	2453919	2454209	1	2453919	2454209
IGLV3-21	NT_011520	+	2445963	2446253	2445963	2446253	1	2445963	2446253
IGLV3-22	NT_011520	+	2437588	2437872	2437588	2437872	1	2437588	2437872
IGLV3-25	NT_011520	+	2420024	2420314	2420024	2420314	1	2420024	2420314
IGLV3-27	NT_011520	+	2401563	2401847	2401563	2401847	1	2401563	2401847
IGLV3-32	NT_011520	+	2327805	2328095	2327805	2328095	1	2327805	2328095
IGLV3-9	NT_011520	+	2552533	2552818	2552533	2552818	1	2552533	2552818
IGLV4-3	NT_011520	+	2604469	2604781	2604469	2604781	1	2604469	2604781
IGLV4-60	NT_011520	+	1907344	1907643	1907344	1907643	1	1907344	1907643
IGLV4-69	NT_011520	+	1776140	1776439	1776140	1776439	1	1776140	1776439
IGLV5-37	NT_011520	+	2172628	2172940	2172628	2172940	1	2172628	2172940
IGLV5-45	NT_011520	+	2121131	2121443	2121131	2121443	1	2121131	2121443
IGLV5-48	NT_011520	+	2098038	2098350	2098038	2098350	1	2098038	2098350
IGLV5-52	NT_011520	+	2063833	2064150	2063833	2064150	1	2063833	2064150
IGLV6-57	NT_011520	+	1940949	1941245	1940949	1941245	1	1940949	1941245
IGLV7-43	NT_011520	+	2140102	2140396	2140102	2140396	1	2140102	2140396
IGLV7-46	NT_011520	+	2114730	2115024	2114730	2115024	1	2114730	2115024
IGLV8-61	NT_011520	+	1843895	1844191	1843895	1844191	1	1843895	1844191
IGLV9-49	NT_011520	+	2088338	2088653	2088338	2088653	1	2088338	2088653
