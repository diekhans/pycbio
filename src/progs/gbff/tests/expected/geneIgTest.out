Warning: Number of CDS features (2) doesn't match number of mRNA features (1) gene: ZNF254 (GeneID:9534)
Warning: Number of CDS features (2) doesn't match number of mRNA features (3) gene: HSFY1 (GeneID:86614)
Warning: can't match CDS to mRNA for: NR_003510.1 gene: HSFY1 (GeneID:86614)
